akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -91,6 +91,24 @@ object Source {
     new Source(scaladsl.Source.fromIterator(() ⇒ f.create().asScala))
 
   /**
+   * Helper to create 'cycled' [[Source]] from sequence of elements.
+   * Example usage:
+   *
+   * {{{
+   * List<Integer> data = new ArrayList<Integer>();
+   * data.add(1);
+   * data.add(2);
+   * data.add(3);
+   * Source.cycle(data);
+   * }}}
+   *
+   * Start a new 'cycled' `Source` from the given elements. The producer stream of elements
+   * will continue infinitely by repeating the same sequence of elements provided during construction.
+   */
+  def cycle[O](elements: java.util.List[O]): javadsl.Source[O, NotUsed] =
```

Comment:
That's not 1:1 with the scala one though - why not `@varargs def cycle[O](elements: O*)`?